### ISSUE TITLE
Loop protection can't be turned off

### ISSUE THREAD

**Issue by TheophileMot** [lukehaas/RunJS#516]
In the options, there's a checkbox for "loop protection". Even when it's turned off, though, the app won't run long loops.

Example:

```
for (let i=0; i<10000; i++) {}
```

Output:

```
RangeError: Potential infinite loop: exceeded 2000 iterations.
```

**Comment by lukehaas** [comment]
@TheophileMot thanks for bringing this to my attention. It will be fixed in the next release.

**Comment by lukehaas** [comment]
This is now resolved in the latest release - version 2.7.5
